src/contexts/data/dataContext.js

```diff
@@ -6,6 +6,7 @@ import {
   useReducer,
   useState,
 } from 'react';
+import { cartReducer } from '../../reducers/cartReducer';
```

Comment:
just a suggestion, if the number of reducer functions increases then you can use an index inside the reducer folder and export all reducer fundtion from there, so the import looks cleaner